src/status_im2/contexts/wallet/create_account/view.cljs

```diff
@@ -1,16 +1,78 @@
 (ns status-im2.contexts.wallet.create-account.view
-  (:require [react-native.core :as rn]
-            [quo2.core :as quo]
-            [re-frame.core :as rf]))
+  (:require
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.core :as quo]
+    [re-frame.core :as rf]
+    [react-native.safe-area :as safe-area]
+    [status-im2.contexts.wallet.common.temp :as temp]
+    [utils.i18n :as i18n]
+    [status-im2.contexts.wallet.create-account.style :as style]))
 
-(defn view
-  []
-  [rn/view
-   {:style {:flex            1
-            :align-items     :center
-            :justify-content :center}}
+(defn- view-internal
+  [{:keys [theme]}]
+  (let [top    (safe-area/get-top)
+        bottom (safe-area/get-bottom)]
+    [rn/view
+     {:style {:flex       1
+              :margin-top top}}
+     [quo/page-nav
```

Comment:
somewhat of a warning, I will ideally merge Ulises' pr refactoring page-nave component today and this should change this api slightly 👍

Reply:
https://github.com/status-im/status-mobile/pull/17031

Reply:
this has been merged 👍